src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Startup.cs

```diff
@@ -8,19 +8,21 @@
 using OrchardCore.Users.TimeZone.Drivers;
 using OrchardCore.Users.TimeZone.Services;
 
-namespace OrchardCore.Users.TimeZone;
-
-[Feature("OrchardCore.Users.TimeZone")]
-public class Startup : StartupBase
+namespace OrchardCore.Users.TimeZone
 {
```

Comment:
Why put back the traditional namespace? Same in src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneService.cs, src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneSelector.cs, src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Models/UserTimeZone.cs, src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Drivers/UserTimeZoneDisplayDriver.cs